packages/app/src/vis-packs/core/complex/ComplexLineVisContainer.tsx

```diff
@@ -34,7 +34,7 @@ function ComplexLineVisContainer(props: VisContainerProps) {
         className={visualizerStyles.dimMapper}
         dims={dims}
         dimMapping={dimMapping}
-        isCached={useValuesInCache(entity)}
+        canSliceFast={useValuesInCache(entity)}
```

Comment:
`isCached` implied "cached in the suspense store" but now consumers are free to use the dimension mapper as they see fit, including without any intermediary cache.